### PR TITLE
Added functionality for clearing AllowedIps field

### PR DESCRIPTION
**Pull request by jicata**
Add functionality for clearing allowed IPs field if Online Contest type is selected on Contest Create - closes https://github.com/SoftUni-Internal/suls-issues/issues/3729

### CODE REVIEW

File: Open Judge System/Web/OJS.Web/Areas/Administration/Views/Contests/Create.cshtml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿@using Resource = Resources.Areas.Administration.Contests.Views.ContestCreate;
+﻿@using OJS.Common.Models
+@using Resource = Resources.Areas.Administration.Contests.Views.ContestCreate;
 @model OJS.Web.Areas.Administration.ViewModels.Contest.ContestAdministrationViewModel
```
Comment (gogo4ds) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Views/Contests/Create.cshtml]: `model` should be first in the file

File: Open Judge System/Web/OJS.Web/Areas/Administration/Views/Contests/Create.cshtml [hunk 2]
```diff
@@ -1,4 +1,5 @@
-﻿@using Resource = Resources.Areas.Administration.Contests.Views.ContestCreate;
+﻿@using OJS.Common.Models
+@using Resource = Resources.Areas.Administration.Contests.Views.ContestCreate;
```
Comment (gogo4ds) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Views/Contests/Create.cshtml]: Split `Resource` from other using

File: Open Judge System/Web/OJS.Web/Areas/Administration/Views/Contests/Create.cshtml [hunk 3]
```diff
@@ -29,5 +30,17 @@
 <script>
     $(function () {
         $('#AllowedIps').data('kendoAutoComplete').value('@GlobalConstants.DefaultPublicIp' + ',');
+
+        $('#Type').on('change',
+            function() {
+                var selectedContestType = $('#Type').data('kendoDropDownList').text().replace(/\s+/g, '');
```
Comment (Innos) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Views/Contests/Create.cshtml]: It's more consistent to check the value, not the text.

File: Open Judge System/Web/OJS.Web/Areas/Administration/Views/Contests/Create.cshtml [hunk 4]
```diff
@@ -29,5 +33,19 @@
...
     $(function () {
         $('#AllowedIps').data('kendoAutoComplete').value('@GlobalConstants.DefaultPublicIp' + ',');
+
+        $('#Type').on('change',
+            function () {
+                var dropdownList = $('#Type').data('kendoDropDownList');
+               // var selectedContestTypeName = dropdownList.text().replace(/\s+/g, '');
+                var selectedContestValue = dropdownList.value();
+                var allowedIps = $('#AllowedIps').data('kendoAutoComplete');
+                debugger;
```
Comment (gogo4ds) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Views/Contests/Create.cshtml]: remove the `debugger`

File: Open Judge System/Web/OJS.Web/Areas/Administration/Views/Contests/Create.cshtml [hunk 5]
```diff
@@ -29,5 +33,19 @@
 <script>
     $(function () {
         $('#AllowedIps').data('kendoAutoComplete').value('@GlobalConstants.DefaultPublicIp' + ',');
+
+        $('#Type').on('change',
+            function () {
+                var dropdownList = $('#Type').data('kendoDropDownList');
+               // var selectedContestTypeName = dropdownList.text().replace(/\s+/g, '');
```
Comment (gogo4ds) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Views/Contests/Create.cshtml]: should this be comment?

**Pull request merged**